package.json

```diff
@@ -25,10 +33,8 @@
     "test.watch": "stencil test --spec --e2e --watchAll",
     "generate": "stencil generate"
   },
-  "dependencies": {
-    "@stencil/core": "^4.7.0"
-  },
   "devDependencies": {
+    "@stencil/core": "^4.7.0",
```

Comment:
My concern with changing this is and `externalRuntime` is that if the developer changes that value, they also need to change Stencil to a hard dependency. I'm not sure that will be very clear to everyone.

Reply:
That would help. We could also add a small note to [this section in the docs](https://stenciljs.com/docs/next/custom-elements#externalruntime)